container-conf/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+build_vars() {
+    : ${build_image_base:=radiasoft/python3}
+    build_is_public=1
+    build_passenv='PYKERN_BRANCH SLICOPS_BRANCH'
+    : ${PYKERN_BRANCH:=} ${SLICOPS_BRANCH:=}
+}
+
+build_as_root() {
+    build_yum install \
+        libXt-devel \
+        libtirpc-devel \
+        motif-devel \
+        nodejs \
+        perl-ExtUtils-Command \
+        perl-FindBin \
+        rpcgen
+}
+
+build_as_run_user() {
+    install_source_bashrc
+    cat > ~/.post_bivio_bashrc <<'EOF'
+export EPICS_BASE=$HOME/.local/epics
+# $EPICS_BASE/startup/EpicsHostArch outputs linux-x86_64; no need to be dynamic here
+export EPICS_HOST_ARCH=linux-x86_64
+bivio_path_insert "$EPICS_BASE/bin/$EPICS_HOST_ARCH"
+export EPICS_PCAS_ROOT=$EPICS_BASE
```

Comment:
We don't need EPICS_PCAS_ROOT or EPICS_DISPLAY_PATH. These are only for running epics display programs.

Reply:
d1aa076

Reply:
f587987